index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Peter Okorafor - Portfolio</title>
+</head>
+
+<body>
+    <section>
+        <header>
+            <nav class="topnav">
+                <a href="#home" class="logo">Peter Okorafor</a>
+
+                <!-- Menu to be implemented in next steps of Microverse projects.
+                <ul class="nav-menu">
+                    <li class="nav-item">
+                        <a href="#" class="logo">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="logo">Works</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="logo">Contact</a>
+                    </li>
+                </ul>
+                -->
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </header>
+    </section>
+
+    <section class="main-content">
```

Comment:
- [ ] kindly align your content well according to figma design.
- Figma Design
![Captureinli](https://user-images.githubusercontent.com/81444338/170432306-b87d7d00-c404-4f38-a19a-c5b9d2fd4769.PNG)
- Your Design
![Captureinl](https://user-images.githubusercontent.com/81444338/170432399-a6904389-9a53-49af-be9a-36669d397231.PNG)

Reply:
For the links to external portfolios, isn't it appropriate to only add links I have a portfolio on (github, linkedin and twitter)?